src/invocable.rs

```diff
@@ -345,6 +347,20 @@ macro_rules! method {
 /// ```
 #[macro_export]
 macro_rules! call {
+    ($cls_name:literal :: $fn_name:literal ($( $args:expr ),*) -> $rett:ty) => {
+        (|| {
+            $crate::RttiSystem::get()
+                .get_class($crate::types::CName::new($cls_name))
+                .ok_or($crate::InvokeError::ClassNotFound($cls_name))?
+                .static_methods()
```

Comment:
this array only contains native static methods, so this code will not work for scripted statics

Reply:
it'd make sense to pull this logic into a method of `RttiSystem::resolve_static_method` rather than putting it in a macro

Reply:
I've added a helper for statics on master: `RttiSystem::resolve_static_by_full_name`

Reply:
switched in 21bcaa9.